data/css/modules/card/_list.scss

```diff
@@ -87,4 +87,16 @@
     .composite &.width-e &__synopsis {
         font-size: 18.8px;  /* 16px spec * 88% because spec was at 110% scale not 125% */
     }
+
+    // Video/audio icon sizing
+    &.width-f,
```

Comment:
You are right. I didn't think it was necessary since the list card won't really ever go down past `&.width-d`, but that should be specified anyway.